### PR TITLE
Menu Tweaks and Rename Notifications to Events

### PR DESCRIPTION
**Pull request by dvaldivia**
- Rename notifications to events.
- Add categories to event destinations
- Menu changes

Screenshot 2023-02-06 at 10 48 16 AM

Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

**Review by pjuarezd — DISMISSED** [review]

**Review by dilverse — DISMISSED** [review]

File: portal-ui/src/screens/Console/NotificationEndpoints/ListNotificationEndpoints.tsx [hunk 1]
```diff
@@ -291,7 +291,7 @@ const ListNotificationEndpoints = ({ classes }: IListNotificationEndpoints) => {
                             navigate(IAM_PAGES.NOTIFICATIONS_ENDPOINTS_ADD);
                           }}
                         >
-                          Add a Notification Target
+                          Add a Event Destination
```
Comment (jinapurapu) [inline portal-ui/src/screens/Console/NotificationEndpoints/ListNotificationEndpoints.tsx:294]:
```suggestion
                          Add an Event Destination
```

**Review by jinapurapu — CHANGES_REQUESTED** [review]

**Review by pjuarezd — APPROVED** [review]

File: portal-ui/src/screens/Console/Console.tsx [hunk 2]
```diff
@@ -70,13 +70,13 @@ const RegisterOperator = React.lazy(() => import("./Support/RegisterOperator"));
...
-  () => import("./NotificationEndpoints/NotificationEndpoints")
+  () => import("./EventDestinations/EventDestinations")
 );
 const AddNotificationEndpoint = React.lazy(
-  () => import("./NotificationEndpoints/AddNotificationEndpoint")
+  () => import("./EventDestinations/AddEventDestination")
 );
 const NotificationTypeSelector = React.lazy(
-  () => import("./NotificationEndpoints/NotificationTypeSelector")
+  () => import("./EventDestinations/EventTypeSelector")
```
Comment (kaankabalak) [inline portal-ui/src/screens/Console/Console.tsx:79]: Would it make sense to rename these constants as well as the constants `IAM_PAGES.NOTIFICATIONS_ENDPOINTS_ADD_SERVICE`, `IAM_PAGES.NOTIFICATIONS_ENDPOINTS_ADD` and `IAM_PAGES.NOTIFICATIONS_ENDPOINTS`?

Comment (dvaldivia) [reply portal-ui/src/screens/Console/Console.tsx:79]: yes, we should also rename those

Comment (dvaldivia) [reply portal-ui/src/screens/Console/Console.tsx]: done @kaankabalak

**Review by allanrogerr — APPROVED** [review]

**Review by kaankabalak — APPROVED** [review]

**Comment by jinapurapu** [comment]
The Add Event Destination screens still describe them as "Notification Endpoints"
Screen Shot 2023-02-07 at 10 39 30 AM

**Comment by dvaldivia** [comment]
@jinapurapu let me address that in a separate PR

**Pull request merged**